linetools/lists/parse.py

```diff
@@ -626,8 +647,11 @@ def parse_morton03(orig=False, tab_fil=None, HIcombine=True):
                         gdZ = gdZ[0]
                     tbl[count]['Z'] = elmZ[gdZ]
                     # Name
-                    tbl[count]['name'] = elmc[gdZ]+ionv[gdi]+' {:d}'.format(
-                        int(tbl[count]['wrest']))
+                    try:
+                        tbl[count]['name'] = elmc[gdZ]+ionv[gdi]+' {:d}'.format(
+                            int(tbl['wrest'][count]))
+                    except UnicodeEncodeError:
+                        pdb.set_trace()
```

Comment:
What would be going on here?

Reply:
Bad characters in the ASCII files pulled from ApJ.